### ISSUE TITLE
Exception: cannot share data context between Swing events

### ISSUE THREAD

**Issue by chenop** [OpherV/gitflow4idea#335]
* **I'm submitting a ...**
  - [ x ] bug report
  - [ ] feature request
  - [ ] puppy => You're not submitting a puppy. I already have one and he's [adorable](https://twitter.com/Opherv/status/974341249099542528)

* **What is the current behavior?**
When Opening Webstorm got the following excpetion:
java.lang.Throwable: cannot share data context between Swing events; initial event count = 569842; current event count = 572763
	at com.intellij.openapi.diagnostic.Logger.error(Logger.java:161)
	at com.intellij.ide.impl.DataManagerImpl$MyDataContext.getData(DataManagerImpl.java:327)
	at com.intellij.openapi.actionSystem.DataContext.getData(DataContext.java:42)
	at com.intellij.openapi.actionSystem.AnActionEvent.getData(AnActionEvent.java:192)
	at com.intellij.openapi.actionSystem.AnActionEvent.getProject(AnActionEvent.java:151)
	at gitflow.actions.StartFeatureAction.actionPerformed(StartFeatureAction.java:36)
	at com.intellij.openapi.actionSystem.ex.ActionUtil.lambda$performActionDumbAware$5(ActionUtil.java:273)
	at com.intellij.util.SlowOperations.lambda$allowSlowOperations$0(SlowOperations.java:77)
	at com.intellij.util.SlowOperations.allowSlowOperations(SlowOperations.java:64)
	at com.intellij.util.SlowOperations.allowSlowOperations(SlowOperations.java:76)
	at com.intellij.openapi.actionSystem.ex.ActionUtil.performActionDumbAware(ActionUtil.java:273)
	at com.intellij.openapi.keymap.impl.IdeKeyEventDispatcher$1.performAction(IdeKeyEventDispatcher.java:616)
	at com.intellij.openapi.keymap.impl.IdeKeyEventDispatcher.lambda$processAction$3(IdeKeyEventDispatcher.java:676)
	at com.intellij.openapi.application.TransactionGuardImpl.performUserActivity(TransactionGuardImpl.java:95)
	at com.intellij.openapi.keymap.impl.IdeKeyEventDispatcher.processAction(IdeKeyEventDispatcher.java:675)
	at com.intellij.openapi.keymap.impl.IdeKeyEventDispatcher.processAction(IdeKeyEventDispatcher.java:626)
	at com.intellij.openapi.keymap.impl.IdeKeyEventDispatcher.processActionOrWaitSecondStroke(IdeKeyEventDispatcher.java:486)
	at com.intellij.openapi.keymap.impl.IdeKeyEventDispatcher.inInitState(IdeKeyEventDispatcher.java:481)
	at com.intellij.openapi.keymap.impl.IdeKeyEventDispatcher.dispatchKeyEvent(IdeKeyEventDispatcher.java:232)
	at com.intellij.ide.IdeEventQueue.dispatchKeyEvent(IdeEventQueue.java:889)
	at com.intellij.ide.IdeEventQueue._dispatchEvent(IdeEventQueue.java:833)
	at com.intellij.ide.IdeEventQueue.lambda$dispatchEvent$8(IdeEventQueue.java:449)
	at com.intellij.openapi.progress.impl.CoreProgressManager.computePrioritized(CoreProgressManager.java:808)
	at com.intellij.ide.IdeEventQueue.lambda$dispatchEvent$9(IdeEventQueue.java:448)
	at com.intellij.openapi.application.impl.ApplicationImpl.runIntendedWriteActionOnCurrentThread(ApplicationImpl.java:781)
	at com.intellij.ide.IdeEventQueue.dispatchEvent(IdeEventQueue.java:502)
	at java.desktop/java.awt.EventDispatchThread.pumpOneEventForFilters(EventDispatchThread.java:203)
	at java.desktop/java.awt.EventDispatchThread.pumpEventsForFilter(EventDispatchThread.java:124)
	at java.desktop/java.awt.EventDispatchThread.pumpEventsForHierarchy(EventDispatchThread.java:113)
	at java.desktop/java.awt.EventDispatchThread.pumpEvents(EventDispatchThread.java:109)
	at java.desktop/java.awt.EventDispatchThread.pumpEvents(EventDispatchThread.java:101)
	at java.desktop/java.awt.EventDispatchThread.run(EventDispatchThread.java:90)

* **Is this a bug? Sorry about that. If so give me explicit details how to reproduce:**

1. Open IntelliJ

* **What is the expected behavior?**
No Exception bubble should appear in webstorm

* **What is the motivation / use case for changing the behavior?**

* **Please tell me about your environment:**
  
  - Gitflow4idea version: 0.7.6
  - Gitflow version: 1.12.3 (AVH Edition)
  
  - *IntelliJ Help -> about > click copy icon and paste here. Should look like this:*
  ~~~~
    Webstorm 2021.1.3
![image](https://user-images.githubusercontent.com/446575/127512546-3f811816-756e-4f81-a399-5a639a1cc8bf.png)

  ~~~~

**Comment by OpherV** [comment]
Thanks for the report.
Does this happen for every project? Specific projects?
If so are any of those public? I need to be able to repro for a fix

**Comment by chenop** [comment]
Hi @OpherV - thanks for taking the time to write back.

Probably happens on my specific project - I cannot share the project publicly but If thats ok I will share it privately with you so you'll be able to reproduce it.
Is that ok? how would you suggest to share it?

Thanks for this awesome plugin!

**Comment by OpherV** [comment]
Let's try with a zip of the project? Hit me at opherv@gmail.com :)

**Comment by chenop** [comment]
@OpherV  Awesome!
Sent you an email.

Appreciate your support!

**Comment by OpherV** [comment]
I can't seem to reproduce this at all even with your project. It might be an OS thing. 
I assume you're running this on a mac? Can you tell me which specific version?

**Comment by OpherV** [comment]
Also, please test if this still happens with the new 0.7.7 version

**Comment by chenop** [comment]
MacOS Big Sur 11.4.
Seems to be solved in 0.7.7.

Thanks for checking - Appreciate the support!
